services/m3aggregator/config/aggregator.go

```diff
@@ -281,40 +285,70 @@ func (c *streamConfiguration) NewStreamOptions(instrumentOpts instrument.Options
 	return opts, nil
 }
 
-// flushConfiguration contains configuration for flushing metrics.
-type flushConfiguration struct {
-	// Flushing function type.
+// flushHandlerConfiguration contains configuration for flushing metrics.
+type flushHandlerConfiguration struct {
+	// Flushing handler type.
 	Type string `yaml:"type" validate:"regexp=(^blackhole$|^logging$|^forward$)"`
```

Comment:
Users are supposed to choose one and only one handler type, which handles the aggregated metrics. There is no chaining involved. 

The reason why only the forward configuration in there because the other types of handlers (`logging` and `blackhole`) do not require any configuration. I could potentially change the configuration struct to something like
```
type flushHandlerConfiguration struct {
  Logging *loggingHandlerConfiguration `yaml:"logging"`
  Blackhole *blackholeHandlerConfiguration `yaml:"blackhole"`
  Forward *forwardHandlerConfiguration `yaml:"forward"`
}  
```
and return an error if no handler configuration or more than one handler configuration if specified, but the `loggingHandlerConfiguration` and `blackholeHandlerConfiguration` will be empty structs, so if you want to use the logging handler, in the yaml file you'll have something like
```
flush:
  logging: {}
```
Which is kinda weird looking as well.